core_api_c_binding/src/lib.rs

```diff
@@ -27,17 +27,21 @@ impl Logger for NullLogger {
 pub unsafe extern "C" fn holochain_new(ptr: *mut Dna) -> *mut Holochain {
     let agent = Agent::from("c_bob".to_string());
 
-    let context = Arc::new(Context::new(
+    let context = Context::new(
         agent,
         Arc::new(Mutex::new(NullLogger {})),
         Arc::new(Mutex::new(SimplePersister::new())),
-    ));
+        STORAGE_PATH,
```

Comment:
This is where a `FileSystemStorage` should be created and put in a `Box`, so that core does not need to know anything about any CAS implementation.